CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Next release
+
+### Added
+
+- Added brakeman and bundle-audit to CI.
+- Added Pry command aliases.
+- Added Rails console methods to switch tenants.
```

Comment:
The current guideline when it comes to the changelog is to only mention changes that are observable by the end user. (not saying I agree with it :)

Reply:
That is indeed the current guideline. I removed the lines.